### ISSUE TITLE
Different "loss" implementation from the paper

### ISSUE THREAD

**Issue by wcycqjy** [wengwanjiang/USDRL-MPCT#2]
Thanks for your great work.

I find the implementation of the **loss function** in `pretrain.py` is different from what is said in the paper. Specifically, the invariance term and the cross-correlation term are not used. Could you provide the implementation?

**Comment by wengwanjiang** [comment]
Hi @wcycqjy ,

Thank you for your kind words and for your careful review of the code! I appreciate you bringing this up.

The implementation actually aligns with the paper, though the naming in the code might be slightly different. Here is where you can find both terms:

* **Invariance Term:** This is implemented via **MSE** and can be found in `pretrain.py` at **Line 193**. I named this variable `similarity` in the code because the essence of the invariance term is to pull together (maximize similarity) the representation vectors of two different data augmentations from the same sample.
* **Cross-correlation Term:** This is called in `pretrain.py` at **Line 207**. The specific mathematical implementation is located in `model/loss.py` at **Line 17** under the function `def xcorr_loss(z1, z2):`.

Additionally, the relative importance of each loss component is controlled by the weight dictionary `ws` (e.g., `sim`, `vac`, and `xcorr`), which is applied at **Line 273** in `pretrain.py`:

```python
ws = {
    'sim': 5,
    'vac': 1.,
    'xcorr': 1e-3
}
```
I hope this helps clarify the implementation! Please feel free to reach out if you have any other questions.

Best regards,

Wanjiang Weng

**Comment by wcycqjy** [comment]
Thank you!
